### ISSUE TITLE
errorlib: context support in sentry is not working properly

### ISSUE THREAD

**Issue by PXke** [inveniosoftware/invenio#2535]
Sentry does now have user context and http context. (i.e: d73bd94b2fbe8666af31dfec25c334c466e6f6df)

In :
```
client.tags_context({'filename': filename}, {'version', CFG_VERSION})
```

client.tags_context takes only two arguments (i.e: http://raven.readthedocs.org/en/latest/usage.html#raven.base.Client.tags_context). 

In : 
```
 client.http_context({'url': req.full_uri,
	 'remote_ip': user_info['remote_ip'],
	 'agent': user_info['agent'],
	 'referer': user_info['referer'],
	 'method': req.method,
	 'query_string': req.args,
	 'headers': dict(req.headers_in),
	 'https': req.is_https(),
	 'env': req.environ})
```

req.environ contains the key  'wsgi.file_wrapper': <built-in method file_wrapper of mod_wsgi.Adapter object at 0x7ffa86c6cf30>

Unfortunately it is not serializable. 

Trying to handle an exception with sentry will then fail.

**Comment by tiborsimko** [comment]
> Can we move this up to milestone 1.2.0 ?

We can, but the PR is still WIP...

**Comment by tiborsimko** [comment]
Note that the context problem was fixed in the meantime in 229d3b24540e2c66d0bae11294d1f07f6ecba15b.

**Comment by PXke** [comment]
Thank you :smile: .
